test/suites/integration/utilization_test.go

```diff
@@ -38,9 +38,9 @@ var _ = Describe("Utilization", Label(debug.NoWatch), Label(debug.NoEvents), fun
 		test.ReplaceRequirements(nodePool,
 			corev1beta1.NodeSelectorRequirementWithMinValues{
 				NodeSelectorRequirement: v1.NodeSelectorRequirement{
-					Key:      v1.LabelInstanceTypeStable,
+					Key:      v1beta1.LabelInstanceCPU,
```

Comment:
Should we just do CMR?

Reply:
Is there a need to specify any additional requirements at all?

Reply:
At the very least we don't want `t` instance types. I don't think theres a specific reason to use CMR, but it seems reasonable to stick with our defaults. This test is really just meant to check bin-packing.

Reply:
Given that, can we just drop all overrides? Or do we need the CPU requirement so we only schedule one per node? If we do need it for this, can we just use anti-affinity to achieve the same thing?